### PR TITLE
Added installation instructions.

### PR DESCRIPTION
**Pull request by blackbaud-brandonhare**

### CODE REVIEW

File: src/app/learn/index.html [hunk 1]
```diff
@@ -1,9 +1,52 @@
...
-  <p>Stache components require SKY UX to use. To learn more, refer to the <a href="https://developer.blackbaud.com/skyux2/learn">SKY UX documentation</a>.</p>
+  <p>
+    Stache components require SKY UX to use. To learn more, refer to the <a href="https://developer.blackbaud.com/skyux2/learn">SKY UX documentation</a>.
+  </p>
+
+  <stache-page-anchor>
+    Start a Stache Project
+  </stache-page-anchor>
+  <p>
+    Starting a Stache project is quick and easy. To get started, run <stache-code>skyux new --template stache</stache-code>. This will generate a new SKY UX project with the Stache template. Once the project generates, run <stache-code>npm install</stache-code> to install the necessary dependencies.
```
Comment (Blackbaud-BrandonJones) [inline src/app/learn/index.html]: I think npm install already runs during this process?   Also might be worth noting, that with the Stache template it also brings in the Stache components?

File: src/app/learn/index.html [hunk 2]
```diff
@@ -1,9 +1,52 @@
...
+  </stache-page-anchor>
+  <p>
+    If you want to add Stache to an established project:
+  </p>
+  <ol>
+    <li>
+      Run <stache-code>npm install --save @blackbaud/stache</stache-code> from your project directory.
+    </li>
+    <li>
+      Add the <stache-code>StacheModule</stache-code> to your app's <stache-code>app-extras.module.ts</stache-code>:
```
Comment (Blackbaud-BrandonJones) [inline src/app/learn/index.html]: maybe explicitly mention to the app-extras.module.ts's imports and exports in order to use the components throughout your app. ?

File: src/app/learn/index.html (hunk 1 above)
Comment (blackbaud-brandonhare) [inline src/app/learn/index.html]: I wasn't sure if it did or not. I can remove that part. Seems unnecessary to mention that it pulls in the components.

File: src/app/learn/index.html (hunk 2 above)
Comment (blackbaud-brandonhare) [inline src/app/learn/index.html]: Sure.

File: src/app/learn/index.html (hunk 1 above)
Comment (Blackbaud-BrandonJones) [inline src/app/learn/index.html]: 👍

**Pull request merged**